app/code/Magento/QuoteGraphQl/etc/schema.graphqls

```diff
@@ -263,6 +263,8 @@ type SelectedShippingMethod {
     method_title: String!
     amount: Money!
     base_amount: Money @deprecated(reason: "The field should not be used on the storefront")
+    price_excl_tax: Money!
```

Comment:
amount and prices can't be used interchangeably
amount should either be removed or renamed to display_price
or price should be change to amount resulting in
amount_excl_tax: Money!
amount__incl_tax: Money!